crates/topos-tce-transport/src/lib.rs

```diff
@@ -117,3 +134,62 @@ pub enum ProtocolEvents {
     /// Stable Sample
     StableSample,
 }
+
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq, Hash)]
+pub struct ValidatorId(H160);
+
+impl ValidatorId {
+    pub fn as_bytes(&self) -> &[u8] {
+        self.0.as_bytes()
+    }
+
+    pub fn address(&self) -> Address {
+        Address::from_str(&format!("0x{}", hex::encode(self.0))).unwrap()
+    }
+}
+
+impl From<H160> for ValidatorId {
+    fn from(address: H160) -> Self {
+        ValidatorId(address)
+    }
+}
+
+impl From<&str> for ValidatorId {
+    fn from(address: &str) -> Self {
+        ValidatorId(H160::from_str(address).expect("Cannot transform to H160"))
```

Comment:
Fair point. I wasn't sure, since:
* If we can't parse it here, the node doesn't function correctly 
* This `from` is being used just in the CLI, and can be removed once we use the `genesis.json` in the CI.